core/arch/arm/plat-mediatek/cros_huk.c

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
What makes this file `plat-mediatek` unique? It looks like this could be a generic function.